src/api_client/constants.js

```diff
@@ -16,3 +16,25 @@ export const GET = 'GET';
 export const POST = 'POST';
 export const PUT = 'PUT';
 export const API_RECONNECT_MAX_RETRY_COUNT = 3;
+
+export const BETANET_NETHASH =
```

Comment:
Do nethashes definitely belong in `api_client`? There's an argument to be made that they're part of the protocol (or at least that they're general enough to warrant being placed in the global constants module).

Reply:
yea true, but is there any other place that can potentially use the nethash?